completions/_nvme

```diff
@@ -617,6 +618,118 @@ _nvme () {
 				;;
 			esac
 			;;
+		(lm)
```

Comment:
@NateThornton Self-reminder these need updating too

Reply:
and while at, also split the tab completions changes into a separate patch. don't forget to update the docs too :)